contrib/IECoreUSD/src/IECoreUSD/VolumeAlgo.cpp

```diff
@@ -81,6 +83,15 @@ using namespace IECoreUSD;
 namespace
 {
 
+static const bool g_forceAssetPathForwardSlash = []() -> bool {
```

Comment:
Let's not duplicate this logic - can we just add `DataAlgo::fromUSD( const SdfAssetPath & )` and call that from VolumeAlgo?

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1426/commits/c775c99c654b72988357554d7627068c58244664